assimilator.py

```diff
@@ -35,8 +35,8 @@ def assimilate(data_type, data_path, data_idName, geo_path, geo_idName, out_path
                 # For row in CSV insert columns as properties in geoJSON
```

Comment:
Sorry if what most of what I am about to comment is quite inane. Is this section is creating a csv which describes the geographical data?

Reply:
Yes - no worries at all @bar-woolf. This script is necessary to generate geoJSON files used in the map? @CMorenoStokoe is absolutely the person you want to talk about this for sure :)

Reply:
No problems @bar-woolf , this function essentially takes CSV data (e.g., deprivation by LSOA area in wales) and 'assimilates' it into boundary files (e.g., for each area boundary in wales the relevant data row in the CSV is entered as 'properties'). These data values allow us to color the map by coloring areas according to these values (generateLayers.py).

Reply:
Thank you @leriomaggio  @CMorenoStokoe

Reply:
@bar-woolf I would save these comments as soon as we close this PR and we will go over reviewing Chris's ;)

Reply:
Marking this as resolved :)